packages/babel-config/src/web.ts

```diff
@@ -99,11 +101,20 @@ export const getWebSideBabelPlugins = (
 }
 
 export const getWebSideOverrides = (
-  { prerender, forVite }: Flags = {
-    prerender: false,
+  { forPrerender, forVite, forRscClient }: Flags = {
+    forPrerender: false,
     forVite: false,
+    forRscClient: false,
   }
 ) => {
+  // Bit of complexity here!
+  // The plugin will modify the Routes file differently based on what context we're building for
+  const routeLoaderOptions: RoutesAutoloaderOptions = {
```

Comment:
Mostly stylistic preference, but I would have kept this inline. It's only used in that one place. And that way we'd get around having to do the (harmless) type import